src/components/badge/Badge.js

```diff
@@ -8,9 +8,9 @@ type Props = {
     /** Child components */
     children: React.Node,
     /** A predefined badge type */
-    type?: 'info' | 'warning' | 'highlight' | 'error' | 'alert' | 'success',
-    /** Adds a class to the component */
     className?: string,
+    /** Adds a class to the component */
```

Comment:
I think you moved the wrong comment, here

Reply:
Yeah I think eslint auto-fix ordering misses comments